blueprints/app/files/package.json

```diff
@@ -27,7 +27,7 @@
     "ember-cli-dependency-checker": "0.0.7",
     "ember-cli-ic-ajax": "0.1.1",
     "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-qunit": "0.1.2",
+    "ember-cli-qunit": "git://github.com/ef4/ember-cli-qunit#hinting",
```

Comment:
No the branch seems correct: https://github.com/ef4/ember-cli-qunit/tree/hinting.  This will be updated to use a released version once https://github.com/ember-cli/ember-cli-qunit/pull/28 lands and is released.